packages/provider-cosmoskit/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Provider } from '@abstract-money/react'
+import {
+  type CosmWasmClient,
+  type SigningCosmWasmClient,
+} from '@cosmjs/cosmwasm-stargate'
+import { useChain } from '@cosmos-kit/react'
+import { useEffect, useMemo, useState } from 'react'
+
+export const cosmosKitProvider = {
+  useCosmWasmClient(args?: { chainName?: string }) {
+    const [client, setClient] = useState<CosmWasmClient | undefined>(undefined)
+    const { getCosmWasmClient: _getCosmWasmClient } = useChain(
+      args?.chainName ?? 'neutron',
```

Comment:
We should probably pull this out into a constant and perhaps have it configurable by the user.

Reply:
Or is the point of this that it just provides a valid chain so the method returns properly?

Reply:
`'neutron'` is rather passed as a hack because otherwise the hook just throws an error when a chain is not defined.

this hack makes it so the `getCosmWasmClient` is undefined until the chain was passed in the args.